dev/core/src/com/google/gwt/core/ext/linker/impl/SelectionScriptLinker.java

```diff
@@ -15,6 +15,8 @@
  */
 package com.google.gwt.core.ext.linker.impl;
 
+import static java.nio.charset.StandardCharsets.UTF_8;
```

Comment:
The existing codebase imports the whole class instead of statically importing this one field, not sure if that matters.